cfme/tests/configure/test_access_control.py

```diff
@@ -2101,11 +2102,91 @@ def test_tenant_visibility_miq_ae_namespaces_all_parents():
     pass
 
 
-@pytest.mark.manual
+def create_user(appliance):
+    """This function will create user, group, and role with all product feature
+    by clicking individually"""
+    COMMON_FEATURES = [
+        "Services",
+        "Compute",
+        "Configuration",
+        "Networks",
+        "Storage",
+        "Control",
+        "Automation",
+        "Monitor",
+        "Help",
+        "API",
+        "Service UI",
+    ]
+
+    FEATURES_511 = COMMON_FEATURES + [
+        "Overview",
+        "Migration",
+        "User Settings",
+        "All VM and Instance Access Rules",
+        "Main Configuration",
+    ]
+
+    FEATURES_510 = COMMON_FEATURES + [
+        "Cloud Intel",
+        "Optimize",
+        "Access Rules for all Virtual Machines",
+    ]
```

Comment:
To avoid duplication of these lists, could you use the roles we have in https://github.com/ManageIQ/integration_tests/blob/7e15cf3c285afc975f3ef6d7671ec16642108968/cfme/roles.py#L286

We should be able to just use the `keys` for some of these dicts.